<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="title">
     <string>LOG IN</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nombre de Usuario:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_usuario">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pass">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_host">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_conectar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONECTAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_error">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(Mensajes)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_usuarios">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>460</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONFIGURACIÓN AUTO LOGIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_conectar_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AUTO LOGIN</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
